e2e-tests/src/main/java/com/dnastack/ddap/frontend/SearchE2eTest.java

```diff
@@ -42,10 +42,10 @@ public void queryPrestoTable() throws IOException {
         ddapPage.getNavBar().goToSearchResources();
         log.info("Authorizing for search resources");
 
-        WebElement accessBtn = driver.findElement(DdapBy.se("explore-resource"));
+        WebElement accessBtn = driver.findElement(DdapBy.se("search-resource"));
         new WebDriverWait(driver, 10)
-                .until(ExpectedConditions.attributeContains(DdapBy.se("explore-resource"), "href", "?resource"));
-        URI authorizeUrl = URI.create(accessBtn.getAttribute("href"));
+                .until(ExpectedConditions.attributeContains(DdapBy.se("search-resource"), "data-href", "?resource"));
+        URI authorizeUrl = URI.create(accessBtn.getAttribute("data-href"));
```

Comment:
It looks like this test is ignored, not sure if there is a specific reason or not. But I just wanted to updated these attributes so it doesn't cause too much trouble if this test is unignored.

Reply:
Feel free to ignore this if the test is ignored, it might as well get deleted. It will need to be updated once the situation is resolved.

Reply:
Agree with @mpanik